network/wsPeer.go

```diff
@@ -534,11 +551,30 @@ func (wp *wsPeer) handleFilterMessage(msg IncomingMessage) {
 	}
 	var digest crypto.Digest
 	copy(digest[:], msg.Data)
-	//wp.net.log.Debugf("add filter %v", digest)
+	wp.net.log.Debugf("add filter %v", digest)
```

Comment:
please restore this comment. It affect the performance of the method, and generally not really helpful